lightning/src/ln/onion_payment.rs

```diff
@@ -3,21 +3,24 @@
 //! Primarily features [`peel_payment_onion`], which allows the decoding of an onion statelessly
 //! and can be used to predict whether we'd accept a payment.
 
-use bitcoin::hashes::Hash;
 use bitcoin::hashes::sha256::Hash as Sha256;
-use bitcoin::secp256k1::{self, PublicKey, Secp256k1};
+use bitcoin::hashes::Hash;
```

Comment:
This might be a bit painful for #3711

https://github.com/lightningdevkit/rust-lightning/pull/3711/files#diff-65f5da7141cd8ea107a9b096e65c8526a4a32dd3e4e4dcbef282eadbbbc5f14d

Reply:
That last commit was just a vanilla rustfmt, wasn't it?

I think (...) that wouldn't be a problem when using the fix script https://github.com/lightningdevkit/rust-lightning/pull/3749#issue-3019056916? Just apply rustfmt to `onion_payment.rs` for each commit of #3711.